lib/ApiRequestor.php

```diff
@@ -141,13 +141,16 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
+        $curlVersion = curl_version();
```

Comment:
have we confirmed that calling this has a minimal impact? We've discussed this before in private but can't remember seeing clear results